example.pypi-private.cfg

```diff
@@ -24,3 +24,8 @@ acl = private
 #   - PP_S3_ACCESS_KEY
 #   - PP_S3_SECRET_KEY
 #   - PP_S3_SESSION_TOKEN
+#
+
+[azure]
+container = mycontainer
+prefix = simple
```

Comment:
Please add a comment about env vars here similar. See also: My other comment in the azure.py module.